core/src/avm2/globals/flash/geom/PerspectiveProjection.as

```diff
@@ -1,44 +1,54 @@
 package flash.geom {
-    import __ruffle__.stub_constructor;
     import __ruffle__.stub_getter;
     import __ruffle__.stub_method;
     import __ruffle__.stub_setter;
+    import flash.display.DisplayObject;
     import flash.geom.Matrix3D;
     import flash.geom.Point;
 
     public class PerspectiveProjection {
-        // Getters are stubbed with what seem to be Flash's default values
+        [Ruffle(NativeAccessible)]
+        private var displayObject: DisplayObject = null;
+
+        [Ruffle(NativeAccessible)]
+        private var fov: Number = 55.0;
```

Comment:
From `TestTransform()` result, `fieldOfView` should be saved and `focalLength` should be computed from it.
`PP(FOV:55, FL:480.25)` assignment to `Sprite.transform` produces `PP(FOV:55, FL:288.15)` (when SWF width is 300).